src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java

```diff
@@ -351,6 +366,8 @@ private void readResources(Document document) {
         // Method
         GrpcMethod grpcMethod = new GrpcMethod(methodname, input, output, method.description());
         grpcMethod.getOptions().add(methodHttpOption);
+        grpcMethod.getOptions().add(requiredMethodSignatureOption);
```

Comment:
@vam-google Does discovery allow for required fields that are not path params? I would have expected so. Do you know? Are we assuming not? (I can modify my scripts to test existing Discovery files.)